src/page-modules/assistant/layout.tsx

```diff
@@ -103,22 +88,20 @@ function AssistantLayout({
 
             <Search
               label={t(PageText.Assistant.search.input.from)}
-              onChange={setSelectedFromFeature}
-              initialFeature={initialFromFeature}
-              selectedItem={selectedFromFeature}
+              onChange={onFromSelected}
+              selectedItem={tripQuery.from ?? undefined}
```

Comment:
Isn't this the same as just passing `tripQuery.from` since it can be undefined?

Reply:
Gah, this is something I'm not very happy with. But difference between `null` and `undefined`. Need to support null for serializing between server and client